code/game/area/Space Station 13 areas.dm

```diff
@@ -119,6 +119,9 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "\improper Large Escape Pod Two"
 	music = "music/escape.ogg"
 
+/area/cryo
```

Comment:
I am really unhappy with the location of this in the middle of the shuttle areas but this file is a mess I need to balkanise anyway

Reply:
Agreed, but this is mostly a patch, we'll balkanise areas soon enough